dash/dash-renderer/src/components/error/menu/DebugMenu.react.js

```diff
@@ -1,156 +1,243 @@
+                            </button>
+                            <button onClick={setDontShowAgain}>
+                                Silence all version notifications
+                            </button>
+                        </div>
+                    ) : null}
+                    <span>v{config.dash_version}</span>
+                    {shouldShowUpgradeNotification(
+                        config.dash_version,
+                        newDashVersion
+                    ) ? (
+                        <button
+                            className='dash-debug-menu__upgrade-button'
+                            onClick={toggleShowUpgradeTooltip}
+                        >
+                            Dash update available - v{newDashVersion}
+                        </button>
+                    ) : null}
+                </div>
+                <div className='dash-debug-menu__divider' />
+                <div className='dash-debug-menu__status'>
+                    Server
+                    <_StatusIcon className='dash-debug-menu__icon' />
+                </div>
             </div>
-        ) : (
-            <div className='dash-debug-menu__content' />
         );
 
-        const alertsLabel =
-            (errCount || !connected) && !opened ? (
-                <div className='dash-debug-alert-label'>
-                    <div className='dash-debug-alert' onClick={toggleErrors}>
-                        {errCount ? (
-                            <div className='dash-debug-error-count'>
-                                {'🛑 ' + errCount}
-                            </div>
-                        ) : null}
-                        {connected ? null : (
-                            <div className='dash-debug-disconnected'>🚫</div>
-                        )}
-                    </div>
-                </div>
-            ) : null;
-
-        const openVariant = opened ? 'open' : 'closed';
-
         return (
             <div>
-                {alertsLabel}
-                <div className={classes('dash-debug-menu__outer', openVariant)}>
+                <div className={classes('dash-debug-menu__outer')}>
+                    {popup == 'errors' && errCount > 0 ? (
+                        <FrontEndErrorContainer
+                            clickHandler={toggleErrors}
+                            errors={errors}
+                            connected={error.backEndConnected}
+                        />
+                    ) : undefined}
                     {menuContent}
                 </div>
-                <div
-                    className={classes('dash-debug-menu', openVariant)}
-                    onClick={() => {
-                        this.setState({opened: !opened});
-                    }}
-                >
-                    <DebugIcon
-                        className={classes('dash-debug-menu__icon', 'debug')}
-                    />
-                </div>
-                <GlobalErrorOverlay
```

Comment:
Is this comoponent not used anymore?

Reply:
Ah yeah I did remove that. It had very little logic in it so I just moved it into the `DebugMenu.react.js`. The only thing I see it doing is concatenating the frontend and the backend errors, which I now do on line 148. Here's the current file: https://github.com/plotly/dash/blob/7c0318749da90424f7de83badeb66b523f6a60a5/dash/dash-renderer/src/components/error/GlobalErrorOverlay.react.js#L18